### PR TITLE
fix: when src of image changed imageLoadedError left true bug

### PR DESCRIPTION
**Pull request by Yhuley**

### CODE REVIEW

File: src/components/avatar/index.tsx [hunk 1]
```diff
@@ -34,6 +34,10 @@ export const Avatar: React.FC<AvatarProps> = ({ className, src, size = 54, alt,
   const fontSize = getFontSize(theme, size);
   const [isImageLoadedError, setImageLoadedError] = React.useState(false);
 
+  React.useEffect(() => {
+    setImageLoadedError(false);
+  }, [src]);
```
Comment (vlacher12) [inline src/components/avatar/index.tsx:39]: Do you need to check if `isImageLoadedError` is true?

**Review by vlacher12 — CHANGES_REQUESTED** [review]

File: src/components/avatar/index.tsx (hunk 1 above)
Comment (vozbrann) [reply src/components/avatar/index.tsx:39]: @vlacher12 
Maybe upload errors should be handled outside of this component. The component can have an onError prop

Comment (Yhuley) [reply src/components/avatar/index.tsx:39]: @vozbrann @vlacher12 so, what is better decision?

Comment (vlacher12) [reply src/components/avatar/index.tsx:39]: what's the purpose of  `onError={() => setImageLoadedError(true)}`?

**Review by vlacher12 — CHANGES_REQUESTED** [review]

File: src/components/avatar/index.tsx (hunk 1 above)
Comment (vozbrann) [reply src/components/avatar/index.tsx:39]: I think that this component should not manage the error state. An error and src should be passed in this component as a prop

Comment (vlacher12) [reply src/components/avatar/index.tsx:39]: Agree

**Comment by vozbrann** [comment]
Component logic should be changed https://github.com/Panenco/panenco-ui/issues/407

**Pull request closed without merging**